tests/integration/indexTest.js

```diff
@@ -208,7 +165,7 @@ describe("Ping", function() {
     });
 });
 
-describe("RateLimiter", function() {
+describe.skip("RateLimiter", function() {
```

Comment:
Oh, it doesn't work because I forgot to change the name. It's now called "Antiflood", and takes a /floodignore command. Here is the patch:

```diff
diff --git a/tests/integration/indexTest.js b/tests/integration/indexTest.js
index 8f34a70..92d270f 100644
--- a/tests/integration/indexTest.js
+++ b/tests/integration/indexTest.js
@@ -165,7 +165,7 @@ describe("Ping", function() {
     });
 });
 
-describe.skip("RateLimiter", function() {
+describe("Antiflood", function() {
     const bot = new TelegramBot();
     const pluginManager = new PluginManager(bot, config, auth);
     pluginManager.loadPlugins(["Ping"]);
@@ -175,9 +175,19 @@ describe.skip("RateLimiter", function() {
         const limit = 50;
         let replies = 0;
 
-        pluginManager.loadPlugins(["RateLimiter"]);
+        pluginManager.loadPlugins(["Antiflood"]);
+        bot.pushMessage({
+            text: "/floodignore 5",
+            from: {
+                id: 1,
+                first_name: 'Root',
+                username: 'root'
+            }
+        });
 
-        const callback = () => replies++;
+        const callback = (arg) => {
+            if (arg.text === "Pong!") replies++;
+        }
         bot.on("_debug_message", callback);
 
         for (let i = 0; i < limit; i++)
@@ -185,7 +195,7 @@ describe.skip("RateLimiter", function() {
 
         setTimeout(function() {
             bot.removeListener("_debug_message", callback);
-            pluginManager.removePlugin("RateLimiter");
+            pluginManager.removePlugin("Antiflood");
             if (replies === 1)
                 done();
             else
```